index.html

```diff
@@ -10,7 +10,49 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+<header>
+    <img class="logo" src="./img/karma-logo.svg" alt="brand logo">
+    <nav class="top-navbar">
+        <div class="nav-item" id="nav-item1">Meet Karma</div>
+        <div class="nav-item" >How it Works</div>
+        <div class="nav-item" >Store</div>
+        <div class="nav-item" >Blog</div>
```

Comment:
Hi Afsha i'm doing a code review to complete the homework :
The format i would suggest for the navbar is this :
<nav class="top-navbar" aria-label="Main navigation">
  <ul>
    <li><a href="#karma" aria-label="Learn about Karma">Meet Karma</a></li>
    <li><a href="#how-it-works" aria-label="Learn how Karma works">How it Works</a></li>
    <li><a href="/store" aria-label="Visit the Karma store">Store</a></li>
    <li><a href="/blog" aria-label="Read the Karma blog">Blog</a></li>
    <li><a href="/help" aria-label="Get help with Karma">Help</a></li>
    <li><a href="/login" aria-label="Login to your Karma account">Login</a></li>
  </ul>
</nav>  
=> In this example, I've used a ul tag to group the navigation items, added aria-label attributes to the nav and a tags to improve accessibility for screen readers, and used descriptive text for each navigation item.
Use appropriate HTML tags: The nav tag should be used to wrap the navigation links, and each navigation item should be an a tag with appropriate attributes.
Add href attributes to each navigation item to create clickable links.
 Use descriptive text for each navigation item that conveys its purpose to users, and avoid using generic or ambiguous text.